adb_graphics/default_specs.yml

```diff
@@ -420,10 +423,15 @@ cref: # Composite reflectivity
     <<: *refl
     ncl_name: REFC_P0_L10_{grid}
     title: Ensemble Composite Reflectivity
+  obs:
+    <<: *refl
+    ncl_name: MergedReflectivityQCComposite_P0_L102_GLL0
+    title: Observed Composite Reflectivity
```

Comment:
I thought it might work okay to add the observations as an additional level to plot here. The script handles ensuring we are pulling it from the appropriate user-provided grib file.